week1/extra-homework/sendJS.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const fs = require('fs');
+function sendJS(res) {
+  res.writeHead(200, { 'Content-Type': 'ext/javascript' });
```

Comment:
```suggestion
  res.writeHead(200, { 'Content-Type': 'text/javascript' });
```

Reply:
😱Sneaky typo. Fixed it. Thanks for the tip